SU-SWS/stanford_paragraph_types
# Updated viewfield paragraph type to *not* have a default ds layout.

# READY FOR REVIEW

# Summary
- Removes the filmstrip default layout from the viewfield paragraph type. Display can then be selected for individual fields.

# Needed By (Date)
- July 27

# Urgency
- none

# Steps to Test

1. On your current development instance, go to admin/structure/paragraphs_type/stanford_viewfield/display
2. Scroll to bottom of page; current layout should be set to "filmstrip"
3. In terminal, cd docroot/modules/stanford/stanford_paragraph_types
4. Checkout this branch, EARTH-273
5. Normally, next step would be "drush features-import-all -y" or even revert feature in Featurs admin page to remove the layout, but it doesn't seem to work when the layout is to be replaced with nothing. Instead, delete "stanford_paragraph_viewfield" from admin/structure/paragraph_type. Note: you will lose any content already in those fields.
6. drush features-import-all -y
7. go back to admin/structure/paragraphs_type/stanford_viewfield/display. Layout should be set to "none".

# Affected versions or modules

# Associated Issues and/or People

# See Also
